lib/tasks/fetch_latest_db.rake

```diff
@@ -6,21 +6,21 @@ task :fetch_latest_db => :environment do
   diaper_backup, partner_backup = fetch_latest_backups
 
   puts "Recreating databases..."
-  system("rails db:drop && rails db:create")
+  system("rails db:migrate db:drop db:create")
```

Comment:
Why a migrate first if we are just going to drop it?

Reply:
Because I had the migrate in the wrong place. :p I realized it almost as soon as I'd pushed but couldn't fix it till now. Fixed!